src/improvements/tasks/eth/002-opcm-upgrade-v200/README.md

```diff
@@ -0,0 +1,43 @@
+# 002-opcm-upgrade-v200: Mainnet OPCM v2.0.0: Unichain
```

Comment:
I've said that this directory is `002-opcm-upgrade-v200`. I'm anticipating that that 000 and 001 will come before this one in other PRs.